bindings/java/pom.xml

```diff
@@ -151,6 +158,11 @@
             <artifactId>dotenv-java</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Add to test scope for some handy debugging, like:

```java
System.out.println(FileUtils.readFileToString(tempDir.resolve(path).toFile(), StandardCharsets.UTF_8));
```

This doesn't affect the formal release.